### PR TITLE
Adding Homepage for daycare user

### LINKED ISSUE #63: Adding Homepage for daycare user
## Expected Behavior

After the daycare user is logged in to the website, he moved to the homepage of the site.
The homepage is the first page that the user exposed after login and described the main details for the daycare (for example the details of the daycare and the orders that are waiting for approval)

## Current Behavior

The dogowner and daycare user are sharing the same homepage that contains the `coming soon...` headline

### Steps to Reproduce

Please provide detailed steps for reproducing the issue.

- [ ] arrange the URL patterns in Hotails and main apps
- [ ] develop logic functionality for daycare users
- [ ] Adding a UI for visualization to logical implementations
- [ ] testing

### PR DESCRIPTION
**Pull request by ErezCohenn**
Signed-off-by: Erez Cohen <erez6356458@gmail.com>

### Desired Outcome

After the daycare user is logged in to the website, he moved to the homepage of the site.
The homepage is the first page that the user exposed after login and described the main details for the daycare such as the details of the daycare and the orders that are waiting for approval

### Implemented Changes

Creating homepage **only** for daycare users, if another types of users will try to reach this site the will redirect back to the login page.

### Connected Issue/Story

Resolves #63 
Depends #56 , #62 , #75, #84

### Definition of Done

- [x] arrange the URL patterns in Hotails and main apps
- [ ] develop logic functionality for daycare users
- [ ] Adding a UI for visualization to logical implementations
- [ ] testing

#### Test coverage

- [x] This PR includes new unit and integration tests to go with the code
  changes
- [ ] The changes in this PR do not require tests

#### Documentation

- [ ] Docs (e.g. `README`s) were updated in this PR
- [ ] A follow-up issue to update official docs has been filed here: [insert issue ID]()
- [x] This PR does not require updating any documentation

### CODE REVIEW

**Comment by jeniaSakirko** [comment]
@ErezCohenn please add "depends on #75"

**Comment by ErezCohenn** [comment]
> @ErezCohenn please add "depends on #75"

why? I don't think this PR depends on static data, the additional information can be also merged after we will have a homepage for daycare.
but I added that this PR depends on #62.
@jeniaSakirko  tell me what you think

**Comment by jeniaSakirko** [comment]
> > @ErezCohenn please add "depends on #75"
> 
> why? I don't think this PR depends on static data, the additional information can be also merged after we will have a homepage for daycare. but I added that this PR depends on #62. @jeniaSakirko tell me what you think

#75 is for the search of the reviews which is needed for the DayCare profile

**Comment by ErezCohenn** [comment]
> > > @ErezCohenn please add "depends on #75"
> > 
> > 
> > why? I don't think this PR depends on static data, the additional information can be also merged after we will have a homepage for daycare. but I added that this PR depends on #62. @jeniaSakirko tell me what you think
> 
> #75 is for the search of the reviews which is needed for the DayCare profile

NP

File: main/views.py
Comment (Omeramsc) [inline main/views.py:33]: If that's the homepage anyway, is there a real need for a profile page? I think it might be confusing (especially in the navbar), unless we have an additional basic homepage for the website (which I recommend...)

File: daycare/test_daycare.py
Comment (Omeramsc) [inline daycare/test_daycare.py:83]: I don't see a real need for both a static_data and non_static_data test. 
if anything, I'd prefer to keep the latter. 
WDYT?

**Comment by Omeramsc** [comment]
Overall LGTM, but I'll wait for your response on my comments.

File: main/views.py
Comment (ErezCohenn) [reply main/views.py:33]: We have a single basic homepage that is shared by both daycare and dog-owner users, so the purpose of the profile page for the daycare as we designed is to update the personal profile details of the daycare user, and this is a future page that we have not implemented now and decided it is not our top priority.

File: daycare/test_daycare.py
Comment (ErezCohenn) [reply daycare/test_daycare.py:83]: I agree with @Omeramsc in that case, in the end, we test for some user, so there is no need to separate the test.
@Yuval-Vino because you implemented these tests I'll leave it to you to take care of it, or if you agree with omer too, I can handle it if you want.
 your choice :)

File: main/views.py
Comment (Omeramsc) [reply main/views.py:33]: If you did not implement it, do not create a route for it.

File: daycare/test_daycare.py
Comment (Omeramsc) [inline daycare/test_daycare.py:81]: would improve readability to assign both sides of the assertion to variable with clear names before the assetion.

File: main/views.py
Comment (ErezCohenn) [reply main/views.py]: Agree
removed the profile implication

File: daycare/test_daycare.py
Comment (Yuval-Vino) [reply daycare/test_daycare.py]: good one! added meaningful names for homepage variables

File: main/views.py
Comment (Omeramsc) [inline main/views.py:22]: do we still need the login_required decorator? if so, who is this last return for

File: daycare/test_daycare.py
Comment (Omeramsc) [inline daycare/test_daycare.py:82]: better to have it like this:
```
daycare_shown_in_homepage = response.context['daycare']
assert daycare_shown_in_homepage == dayCare

images_shown_in_homepage = set(response.context['images'])
assert images_shown_in_homepage == set(Image.get_images_by_daycare_id(daycare_id=dayCare.id))
```

I'd also put into a variable the other side of the equation.

File: main/views.py
Comment (ErezCohenn) [reply main/views.py]: Your right, we only have daycare and dog-owner users type so the last return is unnecessary.
fixed

File: daycare/test_daycare.py
Comment (ErezCohenn) [reply daycare/test_daycare.py]: Fixed

**Review by Yuval-Vino — APPROVED** [review]
LGTM!

File: daycare/test_daycare.py
Comment (Omeramsc) [inline daycare/test_daycare.py:84]: by "other side of the equation" I meant `set(Image.get_images_by_daycare_id(daycare_id=dayCare.id))` should also be put as a variable (especially when it is used in both assertions)

**Review by Omeramsc — APPROVED** [review]
Fix the comment I left,
**After** you do, you may merge the PR by yourself.
good job!

File: daycare/test_daycare.py
Comment (ErezCohenn) [reply daycare/test_daycare.py]: Fixed

**Pull request merged**
